back/src/database/SpotSQL.js

```diff
@@ -20,53 +21,44 @@ async function saveSpot(newSpot){
         text: 'DELETE from spots.spots where spot_id = $1;',
         values: [newSpot.spotId]
     };
+
+    if(utility.isEmpty(newSpot.spotName)){
+        error(fileLabel,"ERROR OBJECT:" + util.inspect(newSpot.spotName,{showHidden: false, depth: null}));
+        error(fileLabel,"Spot Name is invalid");
+        return {"success":false,"data":"Spot Name is invalid"};
+    } else if(utility.isEmpty(newSpot.spotType)){
+        error(fileLabel,"ERROR OBJECT:" + util.inspect(newSpot.spotName,{showHidden: false, depth: null}));
+        error(fileLabel,"Spot Type is invalid");
+        return {"success":false,"data":"Spot Type is invalid"};
+    }
+
     const client = await pool.connect();
     //TODO: NULL CHECK
```

Comment:
Don't need this line anymore. You're doing the NULL/Undefined check with the isEmpty function